trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
```

Comment:
- [ ] It would be nice if you make your readme file more professional by organizing its Getting Started section because your Getting Started section should contain those commands which are important to run your project on the local machine and the Key 👍 I have made the necessary changes in the readme file below ⬇️ you can use that in place of your Readme file 😊

````
# 📗 Table of Contents

- [📖 About the Project](#about-project)
  - [🛠 Built With](#built-with)
    - [Tech Stack](#tech-stack)
    - [Key Features](#key-features)
- [💻 Getting Started](#getting-started)
  - [Setup](#setup)
  - [Prerequisites](#prerequisites)
  - [Install](#install)
  - [Usage](#usage)
  - [Run tests](#run-tests)
- [👥 Authors](#authors)
- [🔭 Future Features](#future-features)
- [🤝 Contributing](#contributing)
- [⭐️ Show your support](#support)
- [🙏 Acknowledgements](#acknowledgements)
- [📝 License](#license)

<!-- PROJECT DESCRIPTION -->

# 📖 [OOP_School_Library] <a name="about-project"></a>

**[OOP_School_Library]** is a tool to record what books are in the library and who borrows them.

## 🛠 Built With <a name="built-with"></a>

- **Languages & Technologies:**
  - Ruby

## Key Features <a name="key-features"></a>

- **[add_Person,Student,and_Teacher_classes]**

<p align="right">(<a href="#readme-top">back to top</a>)</p>

## Getting Started

To get a local copy up and running follow these simple example steps.

### Prerequisites

- Git/GitHub;
- Optional ( IDE installed in your machine ) - recommended IDE: Vscode;
- Ruby Installed in your machine;

### Setup

- To clone this repo:
  - Open the Terminal(Mac/Linux) or the Cmd (Windows);
  - Navigate to where you want to paste the Repo folder;
  - Run the following commands
```
git clone https://github.com/Rhaegar121/OOP-School-Library.git
cd OOP-School-Library
```


### Install
```
  bundle install
```  

### Usage
```
irb
```
### Run tests
```
rspec spec
```


<p align="right">(<a href="#readme-top">back to top</a>)</p>

## Authors

👤 **Kaung Myat Kyaw**

- GitHub: [@githubhandle](https://github.com/Rhaegar121)
- Twitter: [@twitterhandle](https://twitter.com/20Rhaegar)
- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kaung-myat-kyaw-391720227)

## 🔭 Future Features <a name="future-features"></a>

- [ ] **["Decorator"_design_pattern]**
- [ ] **[set_up_associations]**
- [ ] **[add_basic_UI]**

<p align="right">(<a href="#readme-top">back to top</a>)</p>

## 🤝 Contributing <a name="contributing"></a>

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).

## ⭐️ Show your support <a name="support"></a>

Give a ⭐️ if you like this project!

<p align="right">(<a href="#readme-top">back to top</a>)</p>

<!-- ACKNOWLEDGEMENTS -->

## 🙏 Acknowledgments <a name="acknowledgements"></a>

I would like to thank you all who support this project.

<p align="right">(<a href="#readme-top">back to top</a>)</p>

## 📝 License

This project is [MIT](./LICENSE) licensed.